### PR TITLE
Add pull_request_template.md.

### PR DESCRIPTION
**Pull request by bendory**
# Changes

Add standard `pull_request_template.md` modeled on pipelines and others.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] ~Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing~ **N/A**
- [ ] ~Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed~ **N/A**
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] ~Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)~ **N/A**
- [ ] ~Release notes contains the string "action required" if the change requires additional action from users switching to the new release~ **N/A**

/kind documentation

# Release Notes

``` release-note
NONE
```

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Submitter Checklist
+
+As the author of this PR, please check off the items in this checklist:
+
+- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
+- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
+- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
+- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
+  functionality, content, code)
+- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
```
Comment (wlynch) [inline .github/pull_request_template.md:16]: You can omit this. We don't track kind labels in this repo (not that we're against them - we just haven't had any use for them so far).

**Review by wlynch — COMMENTED** [review]
One minor thing, otherwise LGTM!

File: .github/pull_request_template.md (hunk 1 above)
Comment (bendory) [reply .github/pull_request_template.md]: Done.

**Pull request merged**